src/cli.yml

```diff
@@ -73,6 +73,35 @@ args:
         default_value: "clamp"
         possible_values: ["clamp", "tile", "mirror", "border"]
         help: Wrap mode for iChannel3
+    - filter0:
+        long: filter0
+        takes_value: true
+        default_value: "mipmap"
+        possible_values: ["scale", "bilinear", "trilinear", "anisotropic", "mipmap"]
```

Comment:
Super nit: let's order these by quality, similar to https://docs.rs/gfx/0.17.1/gfx/texture/enum.FilterMethod.html

Reply:
done.  want me to update the wrap value in this PR or make a new one?

Reply:
Let's do a separate PR for default wrap value. Thanks!